### ISSUE TITLE
Unable to display the picture

### ISSUE THREAD

**Issue by dmgeurts-mm** [teicee/freeipa-widget-picture#3]
FreeIPA v4.10.1

The avatar image no longer shows in the FreeIPA UI, instead the alt text is shown "[ unable to display the picture ]". It's unlikely to be a permissions issue as if the default IPA admin user sets the avatar the same error is displayed after saving user settings.

If I access the URL for the image the browser shows what looks like base64 code (it shows the text string, not the image). The html element in question is: 

```html
<img name="jpegphoto" style="max-height: 240px; opacity: 1;" class="form-control-static img-thumbnail" alt="[ unable to display the picture ]" src="blob:https://ipa1.domain.com/5418beca-bb02-4d39-a081-3b8387e3400c">
```

**Comment by dmgeurts-mm** [comment]
Just found the solution:

`sudo cp /usr/lib/python3.10/site-packages/ipaserver/plugins/user_jpegphoto.py /usr/lib/python3.11/site-packages/ipaserver/plugins/`  
`sudo ipactl restart`
